controller/sync_strats/rtx.go

```diff
@@ -159,7 +154,7 @@ func (m *routerTxMap) Range(callback func(entries *RouterSender) bool) {
 			pfxlog.Logger().Errorf("could not convert edge router entry got %t: %v", value, value)
 		}
 
```

Comment:
Returning false short circuits rest of iteration. This meant we would only send to first router